scripts/dist-git/setup_git_package

```diff
@@ -24,67 +23,49 @@ fi
 VERBOSE=0
 TEST=
 IGNORE=
+AUTHOR="Fedora Release Engineering <rel-eng@lists.fedoraproject.org>"
 GIT_SSH_URL="ssh://localhost"
 
 Usage() {
     cat <<EOF
 Usage:
-    $0 [OPTIONS] <package_name>
+    $0 <package_name>
 
     Creates a new repo for <package_name>
 
 Options:
     -h,--help			This help message
-    --email-domain DOMAIN       Email domain for git hooks.maildomain
-    --pkg-owner-emails EMAILS   Comma separated list of emails for
-                                git hooks.mailinglist
 EOF
 }
 
-# fail with no arguments
-if [ $# -eq 0 ]; then
+if [ $# -gt 2 ]; then
     Usage
     exit -1
 fi
 
-OPTS=$(getopt -o h -l help -l email-domain: -l pkg-owner-emails: -l default-branch-author: -- "$@")
-if [ $? != 0 ]
-then
-    exit 1
-fi
-
-eval set -- "$OPTS"
-
-while true ; do
+# parse the arguments
+while [ -n "$1" ] ; do
     case "$1" in
-        -h | --help) Usage; exit 0;;
-        --email-domain) EMAIL_DOMAIN=$2; shift 2;;
-        --pkg-owner-emails) PKG_OWNER_EMAILS=$2; shift 2;;
-        --default-branch-author) AUTHOR=$2; shift 2;;
-        --) shift; break;;
+	-h | --help )
+	    Usage
+	    exit 0
+	    ;;
+
+	* )
+	    PACKAGE="$1"
+	    ;;
     esac
+    shift
 done
 
-# fail when more or none packages are specified
-if ! [ $# -eq 1 ]; then
+# I hate shell scripting.  I'm sure the above is totally wrong
```

Comment:
The 'getopt' is valid common way, with `eval set -- "$ARGS"` too (to ensure you have `--` within "$@").  Then you can do `for arg; do ... ; done` or do `shift` explicitly.